Simplified/NYPLAnnotations.swift

```diff
@@ -439,7 +441,7 @@ final class NYPLAnnotations: NSObject {
     }
 
     getServerBookmarks(forBook: id, atURL: annotationUrl) { bookmarks in
-      completionHandler(bookmarks
+      completionHandler(bookmarks?
```

Comment:
This is a small thing, but `X.filter({F}).first` is equivalent to the faster `X.first { F }`. I'd prefer the latter form to avoid constructing a new array needlessly: `first` with the block argument will return the first result that matches immediately.

Reply:
That's great to know, thanks. For this particular instance, instead I decided to delete the entire method. It it no longer being used. It was designed as part of the following logic:

In order to update a bookmark with its server-given annotation ID:
- Upload a bookmark
- Download all bookmarks for the book
- Filter for this bookmark and re-create the object with the ID

This is now accomplished in other methods by:
- Upload a bookmark
- Create the object with the ID present in the response of the upload request